buku

```diff
@@ -190,6 +190,9 @@ def getPageResp(url, fullurl=False):
     Returns: connection, HTTP(S) GET response
     """
 
+    urlconn = None
+
+    # is this really necessary...? Why only spaces ?
```

Comment:
Introduced in commits https://github.com/jarun/Buku/commit/ea02856bc108ea037dce6707fb0de8a323cdfc35 and https://github.com/jarun/Buku/commit/a56fa0381db566a49678f2d5c5f0940608987574.

I can check and get back on the particular URL if you insist on diving further into it.